### ISSUE TITLE
Resulst Return no Results

### ISSUE THREAD

**Issue by tradesouthwest** [WPTRT/ns-theme-check#54]
Just downloaded and your commit notes show last update was two hours ago, so hopefully you already caught this one. I get absolutely no dialogue after the progress bar has shown completion of theme scan. See two images 
1 - old school theme checker
2 - ns-theme-check
![screenshot-larryjudd2017-05-26-09-58-55](https://cloud.githubusercontent.com/assets/2350010/26504727/9b79a426-41fa-11e7-83ed-929ea97e9bf7.png)
![screenshot-larryjudd2017-05-26-10-00-ns](https://cloud.githubusercontent.com/assets/2350010/26504740/a1863eec-41fa-11e7-8b79-31887d29f04f.png)

**Comment by dingo-d** [comment]
Appeal is a theme from ThemeForest? I am testing the latest version and it all seems fine to me. Does this happen when you try to check one of the default themes (twentyseventeen for instance)?

**Comment by grappler** [comment]
The plugin is not installed correctly. Please check the installation guide: https://github.com/WPTRT/ns-theme-check#installing

**Comment by tradesouthwest** [comment]
@grappler When I installed (correctly) the version from this repository, it came up blank from results. When I used an older version from a download of zip which, I downloaded about two weeks ago, everything ran perfectly.

Thanks for the fast response... my resolve is that it might have been downloaded---which I should be forking but I have too many WP themes---at or during the time [you] were updating the commit. I say this because the notation for last commit on several files was "2 mintues ago" or something like that. Assuming that was my issue, everything works fine now.

@dingo-d No. Appeal is a Tradesouthwest Theme, in production.
